### ISSUE TITLE
Rename the FLow attributes `summed_max` and `summed_min`

### ISSUE THREAD

**Issue by uvchik** [oemof/oemof-solph#790]
The `FLow` attributes `summed_max` and `summed_min` can be used to define the minimum or maximum amount of energy of a Flow summarised over the whole period. Due to the Investment Mode these attributes are relative numbers and will be multiplied with the `nominal_value` or the `investment_variable` to get the absolute bound. 

```
maximum_energy = summed_max * nominal_value
maximum_energy = summed_max * investment_variable
```

In most cases the `nominal_value` is something like the installed capacity, therefore `summed_max` is something like the maximum full load time and the unit is a time unit e.g. `nominal_value`: [W] and `summed_max`: [h] the resulting maximum energy bound will have the unit: [Wh].

Related to [this forum post](https://forum.openmod.org/t/can-the-demand-be-optimized-in-oemof-solph-sink/2814)  I got the impression that the name `summed_max` is misleading and should be renamed to something like `max_nominal_time`. 

I do not really like `max_nominal_time` though.
@oemof-developer do you have any ideas? What do you think about this issue?

**Comment by jnnr** [comment]
How about `max_full_load_time` (not hours, because time may be other units, e.g. seconds)?

**Comment by jnnr** [comment]
To be precise, it would have to be `max_yearly_capacity_factor`.

Capacity factors are generally unitless, though. The quantity that we want to rename has units of hours/minutes/etc.

**Comment by jokochems** [comment]
> To be precise, it would have to be `max_yearly_capacity_factor`.
> 
> Capacity factors are generally unitless, though. The quantity that we want to rename has units of hours/minutes/etc.

You are right. We have a time unit here. The `yearly` in turn depends on your simulation timeframe. Once you simulate multiple years (not using multiple periods), it will be an average value since `summed_max` and `summed_mins` are scalars, not iterables, wouldn't it?

**Comment by uvchik** [comment]
By now it is not possible to define multi-periods with summed_max.

I like `max_capacity_factor`. I think it is obvious that it is a factor for the whole optimisation period. 

Even though it might be interesting to make it possible to define a `max_capacity_factor` for e.g. each quarter of the whole period. But this is another issue.

**Comment by uvchik** [comment]
> To be precise, it would have to be `max_yearly_capacity_factor`.

As we are not bound to a year it has to be `max_periodical_capacity_factor`, but I still would vote for `max_capacity_factor`.

**Comment by p-snft** [comment]
Implemented in #815.
